backend/models/ticket.js

```diff
@@ -589,16 +589,23 @@ class Ticket {
 			)
 		]);
 
+		const results = [];
+
 		tickets.forEach(ticket => {
 			if (convertCoords) {
 				this.toDirectXCoords(ticket);
 			}
 
 			const timestamp =  lastUpdated.find(updatedTimestamp =>  utils.uuidToString(updatedTimestamp._id) === ticket._id);
 			ticket.lastUpdated =  (timestamp || {}).lastUpdated || ticket.created;
+
+			if(!updatedSince || updatedSince && ticket.lastUpdated >= updatedSince) {
```

Comment:
can we not add this onto the query instead of filtering it in the js?

Reply:
@sanmont3drepo should just be a query like `$or: [ created: {$gt: x}, comments.created: $gt:x]`?